index.html

```diff
@@ -0,0 +1,268 @@
+</head>
+<body>
+   <header class="main-header">
+    <div class="header-container">   
+    <a href="" class="logo text-color">Solomon</a>
+    <nav>
+        <span class="material-symbols-outlined text-color">
+            menu
+            </span>
+    </nav>
+    </div> 
+   </header>
+   <section class="section-1">
+       <div class="heading-1-container">
+            <h1 class="heading-1 margin-24">Hi there,</h1>
+            <h1 class="heading-1 margin-24">Solomon</h1>
+       </div>
+       <h2 class="heading-2 text-color margin-24">I'm a software Developer</h2>
+       <p class="section-par text-color margin-24">I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+    see and have a project you need coded,
+    don’t hestiate to contact me.</p>
+    
+        <div class="image-0">
+          <img src="socialmedia/design.png" alt="image">
+        </div>
+
+   </section>
+
+
+   <section class="section-2">
+     <div class="gridd">
+      <div class="my-recent-works">
+       <div class="H1">
+         <h1 > MY RECENT WORK</h1>
+       </div>
+      <div class="line">
+      <img  src="image/line.png" alt="line">
+      </div>
+      <div class="main-image">
+        <img src="image/image1.png" alt="main image">
+      </div>
+      <div class="H2">
+        <h2 > Multi-post Stories</h2>
+      </div>
+       <div class="p2">
+        <p>A daily selection of privately personalized reads;
+         no accounts or sign-ups required. has been the industry's standard
+         dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+       </div>
+       <div>
+      <ul class="buttons-0">
+        <li> <button class="btn" type="button">CSS</button></li>
+        <li> <button class="btn" type="button">HTML</button></li>
+        <li> <button  class="btn" type="button">Bootstrap</button></li>
+        <li> <button class="btn" type="button">Ruby</button></li>
+      </ul>
+      </div>
+      <div class="see-project">
+        <a class="seep" href="#"> see project</a>
+      </div>
+      </div>
+ 
+
+
+
+      <div class="profesional-art-1">
+        <div class="H3">
+        <h2> Profesional Art Printing Data</h2>
+        </div>
+        <div class="p2">
+        <p>A daily selection of privately personalized reads;
+          no accounts or sign-ups required.
+          has been the industry's standard</p>
+          </div>
+        <div class="button-0">
+          <ul class="buttons-0">
+            <li> <button class="btn" type="button">HTML</button></li>
+            <li> <button class="btn" type="button">Bootstrap</button></li>
+            <li> <button class="btn" type="button">Ruby</button></li>
+          </ul>
+        </div>
+        <div class="see-project-1">
+          <a class="project" href="#"> see project</a>
+        </div>
+        </div>
```

Comment:
- [ ] Kindly remove the margin-left: 20px; because it's not exist on the Figma design.
- [ ] Kindly make between each project some `margin-bottom` to match the Figma design.
- [ ] Kindly change your paragraph color to white to match the Figma design.
- [ ] Kindly add to each project technology some padding and make them transparent to match the Figma design.

Make these changes on each project you have

Your design
![image](https://user-images.githubusercontent.com/35535130/166127797-3ae1adf1-8b6d-40e0-832c-7c8b403c2da2.png)

Figma design
![image](https://user-images.githubusercontent.com/35535130/166127945-6efde6d1-4654-4ae1-8d6a-df73f75a7495.png)